ext/standard/tests/array/compact.phpt

```diff
@@ -11,6 +11,12 @@ $location_vars = array("c\\u0327ity", "state");
 
 $result = compact("event", $location_vars);
 var_dump($result);
+
+$result = compact(true);
+$foo = 'bar';
+$bar = 'baz';
+$result = compact($foo, [42]);
```

Comment:
will give `['bar' => 'baz']` right? a `var_dump($result);` might be useful?

Reply:
done